docs/reference/code/wethr-recipe.yaml

```
@@ -0,0 +1,44 @@
+# Example recipe for anatomical breakdown
+
+name: wethr
+summary: Command line weather tool.
+description: |
+  Get current weather
+
+adopt-info: wethr
+base: core22
+grade: stable
+confinement: strict
+
+architectures:
+  - build-on: amd64
+  - build-on: armhf
+  - build-on: arm64
+
+apps:
+  wethr:
+    command: bin/wethr
+    plugs:
+      - network
+
+parts:
+  wethr:
+    plugin: npm
+    npm-include-node: true
+    npm-node-version: "10.14.1"
+    source: https://github.com/twobucks/wethr.git
+    override-pull: |
+      Snapcraftctl pull
```

lowercase and use the new syntax:

```suggestion
      craftctl pull
```